### PR TITLE
more details for python

### PR DESCRIPTION
**Pull request by rtlanceroad**
Hi, lucy
1. According to your latest commit 69664bb98950dd6316ceb7e61a126cb0fcd99478 , sys.stdout.detach() is only provided by python3, not python2.
2. I know Arch and Gentoo already switched into python3, but some distros like Crux Linux still use python2 as default, I guess your are a gentoo user, that's why shebang only provides python(since python3 is default for you).
3. Changing shebang in python scripts is a bad idea, at least I think so, so i only changed README.

### CODE REVIEW

**Comment by lucy** [comment]
fixed

**Comment by rtlanceroad** [comment]
3Q. btw, for the irrelevant question,  what's the anime character of your github avatar and which anime? I am curious.

**Comment by lucy** [comment]
tewi is a touhou
